entrypoint.sh

```diff
@@ -31,19 +31,4 @@ esac
 JWT=$(cat $JWT_PATH)
 curl -X POST "http://my.dappnode/data-send?key=jwt&data=${JWT}"
 
-exec geth \
-  --http --http.addr 0.0.0.0 \
-  --http.corsdomain "*" \
-  --http.vhosts "*" \
-  --syncmode ${SYNCMODE:-snap} \
-  --ws \
-  --ws.origins "*" \
-  --ws.addr 0.0.0.0 \
-  --port ${P2P_PORT} \
-  --metrics \
-  --metrics.addr 0.0.0.0 \
-  --authrpc.addr 0.0.0.0 \
```

Comment:
Don't you need to add the `authrpc` settings to the envs too?

Reply:
Yeah, but those are set in Dockerfile.

Reply:
Okay, you are right